plugins/inputs/statsd/statsd.go

```diff
@@ -67,7 +69,12 @@ type Statsd struct {
 	MetricSeparator string
 	// This flag enables parsing of tags in the dogstatsd extension to the
 	// statsd protocol (http://docs.datadoghq.com/guides/dogstatsd/)
-	ParseDataDogTags bool
+	ParseDataDogTags bool // depreciated in 1.10; use datadog_extensions
+
+	// Parses extensions to statsd in the datadog statsd format
+	// currently supports metrics and datadog tags.
+	// http://docs.datadoghq.com/guides/dogstatsd/
+	DataDogExtensions bool
```

Comment:
Can you add a toml struct tag?  I wasn't sure if this would work with this spelling or only with "data_dog_extensions", turns out it works with both but I think it would be best to be explicit and anyway we probably don't want both forms.
```
`toml:"datadog_extensions"`
````